conda_build/build.py

```diff
@@ -736,6 +736,30 @@ def copy_readme(m):
                   "as README.md and README.rst", file=sys.stderr)
 
 
+def jsonify_info_yamls(m):
+    iyd = "info_yaml.d"
+    ijd = "info_json.d"
```

Comment:
I think `info/extra.d/*.json` would be better than `info/info_json.d/*.json`. I think you mentioned that `extra` implies the contents aren't as important. To me, `extra` means more "overflow" for information that's not more-strictly encoded in the files directly in the `info/` directory. A contrived scenario might be if someone creates an `info/info_json.d/index.json` file? How would a sophisticated yet new-to-the-ecoystem user know how to interpret that? If it were me, I think I'd interpret that `info/index.json` is really the source of truth and `info/extra.d/index.json` is something else. Perhaps we can find a better word than `extra` though?